_posts/2021-10-04-openttd-12-0.md

```diff
@@ -0,0 +1,27 @@
+---
+title: OpenTTD 12.0
+author: TrueBrain
+---
+
+The time is finally here: 12.0 is officially released!
+
+12.0 is our "multiplayer update", where we make playing together as easy as possible.
```

Comment:
Tiny improvement:
```suggestion
12.0 is our “multiplayer update”, where we make playing together as easy as possible.
```

Reply:
hihi, personally I do not like those quotes, so I solved it differently: just remove them all together :D

As always, tnx a lot for the feedback, much appreciated :)